app/Importer/MimeTypes.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Importer;
+
+class MimeTypes
```

Comment:
This is another one that is only used once - so we _could_ inline it above if we wanted. But if you have plans or ideas on how to re-use this file elsewhere in the system in the future then I'm fine with it being a separate file. Again, not a dealbreaker for me. (Just, when we're troubleshooting, we often don't like to have to be flipping back and forth between a bunch of files - and we do a *LOT* of troubleshooting).